### PR TITLE
fixing the main readme file

### PR DESCRIPTION
**Pull request by arashb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,39 @@
-# Larq Compute Engine Guide
-
+# Larq Compute Engine
 Larq Compute Engine (LCE) is a highly optimized inference engine for deploying
-neural networks with exteremely low-precision weights and activations,
--such as Binarized Neural Networks (BNNs).
-LCE provides a collection of hand-optimized [TensorFlow](https://www.tensorflow.org/) 
-and [TensorFlow Lite](https://www.tensorflow.org/lite)
-custom ops for specific instruction sets, developed in inline assembly or in C++ 
+neural networks with exteremely low-precision weights and activations, such as
```
Comment (koenhelwegen) [inline README.md]: Perhaps this is a bit verbose? How about just
```suggestion
extremely quantized neural networks, such as
```

File: README.md [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+Binarized Neural Networks (BNNs). It currently supports various mobile platforms
+and has been benchmarked on Pixel 1 phone and Raspberry Pi.
```
Comment (koenhelwegen) [inline README.md]:
```suggestion
and has been benchmarked on a Pixel 1 phone and a Raspberry Pi.
```

File: README.md [hunk 3]
```diff
@@ -1,31 +1,39 @@
...
-  and the Raspberry Pi.
-- Thread parallelism support in LCE is essential for modern mobile devices with
-  multi-core CPUs.
-- A collection of Larq pre-trained BNN models for common machine learning tasks
-  is available in [Larq Zoo](https://github.com/larq/zoo)
-  and can be used seamlessly with LCE.
-- LCE provides a custom [MLIR-based model converter](./docs/mlir_converter.md) which
-  is fully compatible with TensorFlow Lite and performs additional
-  network level optimizations for Larq models.
+- **effortless End-to-end integration** from training to deployment:
```
Comment (koenhelwegen) [inline README.md]:
```suggestion
- **Effortless end-to-end integration** from training to deployment:
```

File: README.md [hunk 4]
```diff
@@ -1,31 +1,39 @@
...
+
+    - Tight integration of LCE with [Larq](https://larq.dev) and
+      TensorFlow provides a smooth end-to-end training and deployment experience.
+
+    - A collection of Larq pre-trained BNN models for common machine learning tasks
+      is available in [Larq Zoo](https://github.com/larq/zoo)
+      and can be used seamlessly with LCE.
```
Comment (koenhelwegen) [inline README.md]:
```suggestion
      and can be used out-of-the-box with LCE.
```

**Comment by arashb** [comment]
> Great! One minor point: we now use `larq.dev` and `github.com/larq/larq` interchangeably. We should probably pick one.

👍 I think make sense to pick the `larq.dev` should I change the larq zoo links to `https://larq.dev/models/` instead of the repo on github as well?

**Comment by leonoverweel** [comment]
Yes I think so, although this link may become `https://docs.larq.dev/zoo` after today, but again if that's the case I'll send a PR to fix it when that happens.

**Pull request merged**